client/src/components/common/applicant-info-nav.vue

```diff
@@ -76,6 +78,7 @@ export default class ApplicantInfoNav extends Vue {
   }
   next () {
     if (this.tab === 3) {
+      this.isloadingSubmission = true
       this.submit()
```

Comment:
Should you await submit()?

Reply:
I think it was 'ok' because the functionality within `submit()` was asynchronous, but i don't see an issue with awaiting it entirely before returning. Probably safer? Will update

Reply:
If the functionality within submit() is async then THIS should await it. It isn't causing an error atm because this is the end of this method (ie, there is nothing after this statement), but fixing it might save a problem later :)